### DISCUSSION TITLE
% Width not working in GitHub Readme (HTML)

### DISCUSSION THREAD

**Discussion by Sabyasachi-Seal** [community/community#16981] [Other Feature Feedback, Questions, & Ideas] [answered]
Before the 19th of May, 2022, My profile looked something like this:-

https://enterprise.githubsupport.com/attachments/token/20NHCxLuEJ7SP7yhZoZKyLMGP/?name=image.png

This was pretty normal and GitHub had no problems with % in widths and heights of photos.

Recently, after the last update and the changelog, the % is no more supported.

This is what my profile looks now.

https://enterprise.githubsupport.com/attachments/token/1VMVA6WCc3oP7SVMYB9iJfSQ4/?name=image.png

I don't know if this is a bug but please tell if this a change that is implemented after or on the 19th of May, 2022 as this will break a lot of designs made by a lot of people on github.

If this change is a feature, could you also tell about a workaround?

**Comment by leandrorodrigues00** [comment]
Hello !!
I had the same problem in several repositories.
Readme markdown (Gifs) is broken after recent GH update.
The solution was to use the HTML `<picture>`,    ~~~~>   #16910

~~~html
<picture>
 <img  src=""/owner/repo/raw/current-branch/.github/exemple.gif"  width="375" height="590em"  />
 </picture>
~~~

That solved my problems!

**Reply by Sabyasachi-Seal** [reply]
I tried doing the same for the pictures. Seems like the pictures are rendering when I preview them, but after I commit, they are not being rendered anymore(black space). I guess GitHub has to fix the stuff themselves.

**Comment by dipree** [comment, marked as answer]
This has been fixed.
